tee-worker/omni-executor/native-task-handler/src/lib.rs

```diff
@@ -396,22 +381,21 @@ async fn handle_native_task<
 							log::error!("Failed to send response");
 						}
 					}
-					let intent_executed_call =
-						parentchain_api_interface::tx().omni_account().intent_completed(
-							omni_account.to_subxt_type(),
-							intent_id,
-							execution_result,
-						);
-					let tx = ctx.transaction_signer.sign(intent_executed_call).await;
-					if rpc_client.submit_tx(&tx).await.is_err() {
-						log::error!("Failed to submit RequestIntent tx");
-						ctx.transaction_signer.update_nonce().await;
-					}
-					return;
+					(execution_result, should_notify_parentchain)
 				},
 			};
 
-			(response_sender, tx)
```

Comment:
Now in the happy path, the `response_sender` is being dropped without sending any message to the channel so it errors out. This is why the test is failing o CI

Reply:
Good catch, thanks!

Reply:
Would you make change directly on this branch? I'm working on another pr

@silva-fj